ui/Screen/Project/Commit.svelte

```diff
@@ -87,6 +313,94 @@
     font-family: var(--typeface-mono-regular);
   }
 
+  .changeset-summary {
+    margin-top: 2rem;
```

Comment:
So far we've been used `px` everywhere, I think switching to relative spacing could be beneficial to make the whole styling a bit more flexible, but we should decide on what's the best way and then do one big change across the whole code base.

If it's too much of a hassle to change it for this PR, would be at least nice to have an issue documenting on how we want to move forward with spacings.

Reply:
I would be in favor of moving to relative spacing, since that is how the app is designed, and it makes it easy to catch mistakes eg. spacing that doesn't fit the system. So I'd rather open an issue for us to move to `rem`, than change this PR and the Commit one too, only to change it back later.

Reply:
Yeah, makes sense, looking forward to the issue.

Reply:
https://github.com/radicle-dev/radicle-upstream/issues/303